docs/contributors/javascript-testing.md

```diff
@@ -18,3 +27,25 @@ Additionally,
 
 -   `test:update` updates the snapshot tests for components, used if you change a component that has tests attached.
 -   `test:watch` keeps watch of files and automatically re-runs tests when things change.
+
+## How to run end-to-end tests
+
+End-to-end tests are implemented in `tests/e2e-tests/specs/`.
+
+Since these drive the user interface, they need to run against a test environment - i.e. a web server running WordPress, Woo and blocks plugin, with a known state/configuration). 
+
+To set up to run e2e tests:
+
+- `npm run build:e2e-test` to build container images used for e2e
+- `npm run wp-env start` to start the test environment
+
+Then, to run the tests:
+
+- `npm run test:e2e` 
+
+When you're iterating on a new test you'll run this repeatedly until your test is just right.
+
+When you're done, you may want to shut down the test environment:
+
+- `npm run wp-env stop` to stop the test environment
```

Comment:
Might be good to link to the [`wp-env` docs] for all the other commands that can be executed?
```suggestion
- `npm run wp-env stop` to stop the test environment

**Note:** There are a number of other `wp-env` commands that would be useful to know. You can see them [here](https://github.com/WordPress/gutenberg/blob/master/packages/env/README.md).
```